aeppl/__init__.py

```diff
@@ -4,7 +4,13 @@
 del get_versions
 
 
-from .logprob import logprob  # isort: split
+from aeppl.logprob import logprob  # isort: split
 
-from .joint_logprob import joint_logprob
-from .printing import latex_pprint, pprint
+from aeppl.joint_logprob import joint_logprob
+from aeppl.printing import latex_pprint, pprint
+
+# isort: off
+# Add optimizations to the DBs
```

Comment:
Is this comment needed?

Reply:
Why wouldn't it be?

Reply:
It was unclear to me if it meant anything

Reply:
Oh, it's just a warning that the import is needed in order to register the mixture optimizations, and it's not there to expose the mixture module at the package level, for example.